astropy/coordinates/builtin_frames/icrs_cirs_transforms.py

```diff
@@ -152,7 +152,7 @@ def gcrs_to_icrs(gcrs_coo, icrs_frame):
     else:
         #compute the distance as just the cartesian sum from moving to the SSB
         #we have to do this because the ERFA functions throw away distance info
-        distance = np.sum((-astrom['eb']*u.au + gcrs_coo.cartesian.xyz.T)**2, -1)**0.5
+        distance = np.sum((astrom['eb']*u.au + gcrs_coo.cartesian.xyz.T)**2, -1)**0.5
```

Comment:
will add the ``axis`` part - I completely agree with you but was just being lazy here.

Re: the transpose: it's needed because ``astrom['eb']`` is sometimes a vector of length equal to the *non-3* dimension of ``xyz``, so the transpose there was the easiest way to get the broadcasting to work.  I think it would also work to reshape the astrom['ab'] to (1,n), but this seems simpler...